src/offchainapi/tests/test_protocol.py

```diff
@@ -570,40 +563,39 @@ def test_sample_command():
     assert obj2.previous_versions == obj.previous_versions
 
 
-def test_parse_handle_request_to_future(signed_json_request, channel, key):
-    fut = channel.parse_handle_request(
-        signed_json_request)
-    res = fut.content
-    res = json.loads(key.verify_message(res))
+async def test_parse_handle_request_to_future(signed_json_request, channel, key):
+    fut = await channel.parse_handle_request(signed_json_request)
```

Comment:
The name "fut" is a little awkward here. Since you are editing this line call it 'response' or something?

Reply:
renamed to `response`